de.flose.Kochbuch.json

```diff
@@ -0,0 +1,98 @@
+{
+    "id": "de.flose.Kochbuch",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.mono6"
+    ],
+    "command": "kochbuch",
+    "finish-args": [
+        "--filesystem=home",
```

Comment:
why does it need this broad home filesystem access?

Reply:
The application allows import and export of recipe files and adding image files to recipes.
Mono winforms doesn't support portals so this is needed to allow the user to select any file in their home directory.